src/OrchardCore.Modules/OrchardCore.Search.Lucene/Migrations.cs

```diff
@@ -49,7 +49,7 @@ public int Create()
                                 }
                                 else
                                 {
-                                    if ((bool)included && !(bool)analyzed)
+                                    if ((bool)included && (analyzed == null || !(bool)analyzed))
```

Comment:
In the above line 43 we check `if (analyzed != null)` so here we know that `analyzed` is null and would not need to be checked at all.

@Skrypt maybe the source of the problem is that line 43 you wanted to check `if (analyzed == null)` instead, which is not the same in term of functionality. If so we would only need to update line 43.

Reply:
I will take a look but that makes sense that I could have made a mistake with `if (analyzed != null)`